dataprep/eda/basic/render.py

```diff
@@ -244,11 +263,12 @@ def hist_viz(
         tools=[],
         y_axis_type=yscale,
     )
+    bottom = 0 if yscale == "linear" or df.empty else df["freq"].min() / 2
```

Comment:
Is yscale = "log" has the same bottom as the yscale = "linear"? 

Another thing is that when I try yscale = "log", the output fig. is the same as yscale = "linear". Is this easy to fix? If not, do not need to fix it in this pr. 
For your reference, the code is:
`plot(df, 'YearBuilt', 'SalePrice', yscale = "log")`
, and the output is:
<img width="477" alt="Screen Shot 2020-04-02 at 12 02 56 AM" src="https://user-images.githubusercontent.com/18078770/78219909-5be5bd80-7475-11ea-8e41-8ba9dcaa5d59.png">

Reply:
1) We can't take the log of 0, so this `bottom` variable is different for log and linear axes. Previously we had bottom=0.01 for all plots, but this is a problem if say for the KDE plot the densities are all smaller than 0.01; this is the reason for the current formula. Feel free to try bottom=0 with the log scale to see what happens, or let me know if you think we should handle this differently.

2) It's easy to make the boxplot support a log scale. Currently the `yscale` parameter is only passed to the histogram plots and KDE plot, so what are all the plots that should support a log scale? If we want to support scatter plots and hexbin plots, we should probably also add a parameter `xscale` for log transforming the xaxis. I think this should be handled in another PR, but let me know what you think.

Reply:
Looks good to me. We can discuss the second issue in the meeting.